src/schema/sources.ts

```diff
@@ -818,8 +947,27 @@ export const resolvers: IResolvers<any, Context> = {
 
       return getSourceById(ctx, info, sourceId);
     },
+    removeMember: async (
+      _,
+      { sourceId, memberId }: RemoveMemberArgs,
+      ctx,
+    ): Promise<GQLEmptyResponse> => {
+      await ensureSourcePermissions(ctx, sourceId, {
+        validateRankAgainstId: memberId,
+        permission: SourcePermissions.MemberRemove,
+      });
+
+      await ctx
```

Comment:
I'm not sure that's the right implementation. Product expects to have some sort of blocked users list. Suggesting to scope it out and work separately on this. @rebelchris

Reply:
@rebelchris let me know if you want this removed and wait until the "banned" user takes place or just keep it as it is.

Reply:
@sshanzel I know we just discussed it, but let's rather remove it for now to avoid confusion later on.
Rather we re-implement it again.

Sorry for the mixed feedback 😅

Reply:
I have removed the function in the schema **but** I commented out the test cases, I think the scenarios written would be a good starting point for writing the feature as the intent would almost likely be the same. Let me know what you think @rebelchris @idoshamun